README.md

```diff
@@ -50,7 +50,9 @@ _Resulting layout:_
 ```html
 <header>
   <h1>Blog 2000</h1>
-  <h2>All articles</h2>
+  <yield to="header">
```

Comment:
So I know that this is influenced by rails. But is `to` the right word here? In my head `target` or something in those lines make more sense.